### ISSUE TITLE
Extend Semantic Annotations

### ISSUE THREAD

**Issue by alessiogambi** [BeamNG/BeamNGpy#47]
If I am not mistaken, the semantic annotations provided by BeamNG uses the same color/label for each type of object. This makes it difficult to distinguish cars when they are close to each other (or one occludes the other) and tracking the movement of objects. Maybe we can have each object a different label and some additional metadata attached to the image. For example, color X, element Y, type=car, model=Z, etc. ?

**Comment by Palculator** [comment]
Yes, this — I think — is called instance annotations. It's something that's been on my TODO list, but I'm afraid there's no ETA right now.
